samples/TcpClient_NarodMon/app/application.cpp

```diff
@@ -51,6 +51,7 @@ bool nmOnReceive(TcpClient& client, char *buf, int size)
 	// debug msg
 	debugf("nmOnReceive");
 	debugf("%s", buf);
+	return true;
```

Comment:
That change should be part of another PR related to fixes in the TcpClient_NarodMon sample.